cmd/kubeadm/app/phases/certs/certs.go

```diff
@@ -165,6 +165,29 @@ func CreatePKIAssets(cfg *kubeadmapi.MasterConfiguration, pkiDir string) error {
 		fmt.Println("[certificates] Generated API server kubelet client certificate and key.")
 	}
 
+	// If the key exists, we should try to load it
+	if pkiutil.CertOrKeyExist(pkiDir, kubeadmconstants.ServiceAccountKeyName) {
+		// Try to load sa.key from the PKI directory
+		apiKey, err := pkiutil.TryLoadKeyFromDisk(pkiDir, kubeadmconstants.ServiceAccountKeyBaseName)
+		if err != nil || apiKey == nil {
```

Comment:
can you remove all nil checks from the `if` checks after `TryLoadKeyFromDisk` in this file.
I think that won't ever happen